packages/flutter/test/widgets/slivers_test.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:collection/collection.dart';
```

Comment:
Are you importing this only for ListEquals? use `listEquals` from flutter/foundation.dart instead.